docs/openapi-qtsp.yaml

```diff
@@ -40,7 +40,7 @@ paths:
             schema:
               $ref: '#/components/schemas/CreateChallengeBody'
```

Comment:
there's still some disalignment: the operation id is `getChallenge` but the payload is named `CreateChallenge` (choose one : - )